apps/web/lib/components/kanban-card.tsx

```diff
@@ -133,62 +142,84 @@ export default function Item(props: any) {
+								/>
+								<span className="text-grey text-normal mr-1">#{item.number}</span>
+								<Link
+									href={`/task/${item.id}`}
+									className="text-black dark:text-white text-normal capitalize mr-2 bg-blue line-clamp-2"
+								>
+									{item.title}
+								</Link>
+								<Priority level={1} />
+							</>
+						) : (
+							<div className="w-56">
+								<TaskInput
+									task={taskEdition.task}
+									initEditMode={true}
+									keepOpen={true}
+									showCombobox={false}
+									autoFocus={true}
+									autoInputSelectText={true}
+									onTaskClick={(e) => {
+										console.log(e);
+									}}
+									onEnterKey={() => {
+										taskEdition.setEditMode(false);
+									}}
+								/>
+							</div>
+						)}
+					</div>
 					<CircularProgress percentage={10} />
 				</div>
-			</div>
-			<div className="flex flex-row justify-between items-center pt-4 h-fit">
-				{item.status === TaskStatus.INPROGRESS ? (
-					<div className="flex flex-row items-center gap-2">
-						<small className="text-grey text-xs text-normal">Live:</small>
-						<p className="text-[#219653] font-medium text-sm">
-							{pad(hours)}:{pad(minutes)}:{pad(seconds)}{' '}
-						</p>
-					</div>
-				) : (
-					<div className="flex flex-row items-center gap-2">
-						<small className="text-grey text-xs text-normal">Worked:</small>
-						<p className="text-black dark:text-white font-medium text-sm">
-							{pad(hours)}:{pad(minutes)}:{pad(seconds)}{' '}
-						</p>
+				<div className="my-2">
+					<HorizontalSeparator />
+				</div>
+				<div className="w-full flex items-center justify-between">
+					<div className="mt-1">
+						{item.status === TaskStatus.INPROGRESS ? (
+							<div className="flex items-center gap-2">
+								<small className="text-grey text-xs text-normal">Live:</small>
+								<p className="text-[#219653] font-medium text-sm">
+									{pad(hours)}:{pad(minutes)}:{pad(seconds)}{' '}
+								</p>
+							</div>
+						) : (
+							<div className="flex items-center gap-2">
+								<small className="text-grey text-xs text-normal">Worked:</small>
+								<p className="text-black dark:text-white font-medium text-sm">
+									{pad(hours)}:{pad(minutes)}:{pad(seconds)}{' '}
+								</p>
+							</div>
+						)}
 					</div>
-				)}
-				<ImageComponent images={taskAssignee} />
-			</div>
-			{item.hasComment && (
-				<div className="flex flex-row items-center justify-center rounded-full w-5 h-5 z-10 bg-[#e5e7eb] dark:bg-[#181920] absolute top-0 right-0">
-					<div className="w-3.5 h-3.5 rounded-full" style={setCommentIconColor(item.hasComment)}></div>
+					<ImageComponent images={taskAssignee} />
+					{/* {item. && (
+						<div className="flex flex-row items-center justify-center rounded-full w-5 h-5 z-10 bg-[#e5e7eb] dark:bg-[#181920] absolute top-0 right-0">
+							<div
+								className="w-3.5 h-3.5 rounded-full"
+								style={setCommentIconColor(item.hasComment)}
```

Comment:
@Anishali2 why this commented out!?

Reply:
Sorry I've forget to uncomment this let me check it again